src/sentry/lang/javascript/sourcemaps.py

```diff
@@ -152,8 +159,73 @@ def sourcemap_to_index(sourcemap):
     return SourceMapIndex(state_list, key_list, src_list, content)
 
 
-def find_source(indexed_sourcemap, lineno, colno):
+def sourcemap_to_index(sourcemap):
+    """
+    Converts a raw sourcemap string to either a SourceMapIndex (basic source map)
+    or IndexedSourceMapIndex (indexed source map w/ "sections")
+    """
+    smap = json.loads(sourcemap)
+
+    if smap.get('sections'):
+        # indexed source map
+        offsets = []
+        maps = []
+        for section in smap.get('sections'):
+            offset = section.get('offset')
+
+            offsets.append((offset.get('line'), offset.get('column')))
+            maps.append(_sourcemap_to_index(section.get('map')))
+
+        return IndexedSourceMapIndex(offsets, maps)
+    else:
+        # standard source map
+        return _sourcemap_to_index(smap)
+
+
+def get_inline_content_sources(sourcemap_index, sourcemap_url):
+    """
+    Returns a list of tuples of (filename, content) for each inline
+    content found in the given source map index. Note that `content`
+    itself is a list of code lines.
+    """
+    out = []
+    if isinstance(sourcemap_index, IndexedSourceMapIndex):
+        for map in sourcemap_index.maps:
+            out = out + get_inline_content_sources(map, sourcemap_url)
```

Comment:
_technically_. Since it's the builtin function `map()` which you're clobbering. To be safe, I'd just call it `smap`.